xml/System.IO/Directory.xml

```diff
@@ -550,7 +550,7 @@ An I/O error occurred.</exception>
   
  This method throws an <xref:System.IO.IOException> if the directory specified in the `path` parameter contains files or subdirectories.  
   
- The `path` parameter is not case-sensitive.  
+ The case-sensitivity of the `path` parameter corresponds to that of the file system on which the code is running. For example, it’s case-insensitive on NTFS (Windows default filesystem) and case-sensitive on ext3 (a Linux filesystem).
```

Comment:
```suggestion
 The case-sensitivity of the `path` parameter corresponds to that of the file system on which the code is running. For example, it's case-insensitive on NTFS (the default Windows file system) and case-sensitive on Linux file systems.
```

Reply:
The wording in the first parentheses is definitely better. I will apply it.

I don’t know if *all* “Linux file systems” (if you can even say that) are case sensitive, but it seems like a reasonable assumption to make here; at least most will surely be case sensitive.